xbmc/pictures/GUIWindowPictures.cpp

```diff
@@ -352,7 +352,8 @@ bool CGUIWindowPictures::ShowPicture(int iItem, bool startSlideShow)
   // look into using OnPlay announce!
   m_slideShowStarted = true;
   CServiceBroker::GetGUI()->GetWindowManager().ActivateWindow(WINDOW_SLIDESHOW);
-
+  if (!startSlideShow)
+    m_viewControl.SetSelectedItem(slideShow.GetCurrentSlide()->GetPath());
```

Comment:
For better readability I would insert a blank line before the `return` statement on next line, like it was before your change.